src/NextcloudCalendar.php

```diff
@@ -0,0 +1,443 @@
+<?php declare(strict_types=1);
+
+namespace Pdsinterop\Flysystem\Adapter;
+
+use League\Flysystem\AdapterInterface;
+use League\Flysystem\Config;
+use OCP\Files\Folder;
+
+use OCA\DAV\CalDAV\CalDavBackend;
+use OCA\DAV\Connector\LegacyDAVACL;
+use OCA\DAV\CalDAV\CalendarRoot;
+use OCA\DAV\Connector\Sabre\Auth;
+use OCA\DAV\Connector\Sabre\ExceptionLoggerPlugin;
+use OCA\DAV\Connector\Sabre\MaintenancePlugin;
+use OCA\DAV\Connector\Sabre\Principal;
+
+
+/**
+ * Filesystem adapter to convert RDF files to and from a default format
+ */
+class NextcloudCalendar implements AdapterInterface
+{
+    private $defaultAcl;
+    private $userId;
+    private $principalUri;
+
+    final public function __construct($userId, $defaultAcl)
+    {
+        $this->userId = $userId;
+        $this->principalUri = "principals/users/" . $this->userId;
+	$this->defaultAcl = $defaultAcl;
```

Comment:
You might want to check the tabs/spaces in this file.

Reply:
fixed, converted all to spaces to comply with PSR